### PR TITLE
Tidying up the README

### PR DESCRIPTION
**Pull request by bethaitman-da**
A few small text changes, making it slightly easier to follow, and trying to make the line breaks more consistent. (I'm also happy to switch to no line breaks, I think that works fine too.)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,10 +1,12 @@
 # Contributing to DAML
 
-Welcome! This document provides a high-level overview of how to contribute to the development of DAML.
+Welcome! This page gives a high-level overview of how to contribute to the development of DAML.
 
-(For information on how to build, test, and work on the codebase, see the [README](./README.md).)
+There are many ways you can contribute beyond coding. For example, you can report problems, report and clarify issues, and write documentation. If you're completely new to open source development, the [Open Source Guides](https://opensource.guide) is a great place to start.
```
Comment (gerolf-da) [inline CONTRIBUTING.md]: What's the difference between "reporting problems" and "reporting issues"?

File: README.md [hunk 2]
```diff
@@ -5,57 +5,70 @@ SPDX-License-Identifier: Apache-2.0
...
+### 1. Clone this repository
 `git clone git@github.com:DACH-NY/daml.git`.
 
-### Setting up the DA Development Environment (also known as `dev-env`)
-dev-env is used to provide dependencies required during build phase, like java,
-bazel, and python for some tooling scripts, while the code itself is built through
-bazel.
+### 2. Set up the DA Development Environment ("`dev-env`")
+
+`dev-env` provides dependencies required during build phase, like Java, Bazel, and Python
```
Comment (gerolf-da) [inline README.md]: missing "the"?
```suggestion
`dev-env` provides dependencies required during the build phase, like Java, Bazel, and Python
```

File: README.md [hunk 3]
```diff
@@ -5,57 +5,70 @@ SPDX-License-Identifier: Apache-2.0
...
+for some tooling scripts. The code itself is built using Bazel.
+
+#### Set up `dev-env` on Linux or macOS
 
-#### Linux and macOS
 1. Install Nix: `bash <(curl https://nixos.org/nix/install)`
-2. Enter dev-env: `eval "$(dev-env/bin/dade assist)"`
+2. Enter `dev-env`: `eval "$(dev-env/bin/dade assist)"`
+
+If you don't want to `dev-env` manually each time using `eval "$(dev-env/bin/dade assist)"`,
```
Comment (gerolf-da) [inline README.md]: Is "to dev-env" a verb or is there an "enter" missing?

File: CONTRIBUTING.md (hunk 1 above)
Comment (bethaitman-da) [reply CONTRIBUTING.md]: from the original - I thought issues were specifically GH issues, and problems more general, but I agree it's not clear

File: README.md (hunk 3 above)
Comment (bethaitman-da) [reply README.md]: Good spot, fixed!

Comment (gerolf-da) [reply README.md]: 👍

**Pull request merged**